core/src/main/scala/caliban/schema/ArgBuilder.scala

```diff
@@ -14,6 +16,8 @@ import magnolia._
 import mercator.Monadic
 import zio.Chunk
 
+import scala.util.control.NonFatal
```

Comment:
nit: can you group this import with the others? I think I use the default IntellIiJ order.

Reply:
Ah, your sorting is correct, it's the old imports that are wrong. Would be good to group them still 😄